src/collection-browser.ts

```diff
@@ -909,7 +920,7 @@ export class CollectionBrowser
   private async fetchFacets() {
     if (!this.fullQuery) return;
 
-    const aggregations = {
+    /* const aggregations = {
```

Comment:
q: why we removed this part?

also instead of commenting it out, maybe we can totally remove this part of the code or atleast add a note why it is still there.

Reply:
Ah, good catch -- I'll remove the old comments here and below.

The context for this removal is that the PPS defaults these aggregation parameters based on the page type being requested (here `search_results` is the default page type used). So we don't need to explicitly request them.

Reply:
In contrast, the other search call that fetches the search results themselves needs to explicitly omit the aggregations because of this defaulting.